src/sentry/dynamic_sampling/rules/helpers/sliding_window.py

```diff
@@ -13,6 +13,28 @@
 # Sentinel value used to mark that an error happened when computing the sliding window sample rate for a specific
 # project.
 SLIDING_WINDOW_CALCULATION_ERROR = "sliding_window_error"
+# We want to keep the entry for 1 hour, so that in case an org is not considered for 1 hour, the system will fall back
+# to the blended sample rate.
+EXECUTED_CACHE_KEY_TTL = 60 * 60 * 1000
+
+
+def generate_sliding_window_executed_cache_key() -> str:
+    return "ds::sliding_window_executed"
+
+
+def mark_sliding_window_executed() -> None:
+    redis_client = get_redis_client_for_ds()
+    cache_key = generate_sliding_window_executed_cache_key()
+
+    redis_client.set(cache_key, 1)
+    redis_client.pexpire(cache_key, EXECUTED_CACHE_KEY_TTL)
```

Comment:
You can pass the expiration as third parameter to `set`. IIRC, it is given in seconds though, so the value should be 3600.

Compare to calls in event_manager, for instance.